libucru: recognize aarch64

Make the same as "arm" internally.

Upstream-Status: Pending

Signed-off-by: joe.slater@windriver.com
[moved to buildroot from openembedded-core]
Signed-off-by: Ben Shelton <benjamin.h.shelton@intel.com>


--- a/configure.ac
+++ b/configure.ac
@@ -77,6 +77,7 @@ AS_CASE([$host_cpu],
 	[alpha*], [ARCHTYPE="alpha"],
 	[ia64], [ARCHTYPE="gcc"],
 	[arm*], [ARCHTYPE="arm"],
+	[aarch64], [ARCHTYPE="arm"],
 	[mips*], [ARCHTYPE="mips"],
 	[tile*], [ARCHTYPE="gcc"],
 	[ARCHTYPE="unknown"]
